<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FeedbackClass</class>
 <widget class="QMainWindow" name="FeedbackClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>383</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>616</width>
    <height>383</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>616</width>
    <height>383</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>忘却的战场-测试版</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Demo.png</normaloff>../Demo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>321</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">QLabel{
	font: 75 12pt &quot;微软雅黑&quot;;
	
	color: rgb(255, 255, 255);
}</string>
         </property>
         <property name="text">
          <string>姓名(选填)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nameEdit">
         <property name="styleSheet">
          <string notr="true">QLineEdit{
	font: 75 12pt &quot;微软雅黑&quot;;
	
	color: rgb(0, 0, 0);
    background-color: rgba(255, 255, 255, 40%)
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">QLabel{
	font: 75 12pt &quot;微软雅黑&quot;;
	
	color: rgb(255, 255, 255);
}</string>
         </property>
         <property name="text">
          <string>电子邮件(选填)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="emailEdit">
         <property name="styleSheet">
          <string notr="true">QLineEdit{
	font: 75 12pt &quot;微软雅黑&quot;;
	
	color: rgb(0, 0, 0);
    background-color: rgba(255, 255, 255, 40%)
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>12</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	font: 75 12pt &quot;微软雅黑&quot;;
	
	color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>反馈内容(必填)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPlainTextEdit{
	font: 75 12pt &quot;微软雅黑&quot;;
	
	color: rgb(0, 0, 0);
    background-color: rgba(255, 255, 255, 40%);
}</string>
       </property>
       <property name="plainText">
        <string/>
       </property>
       <property name="backgroundVisible">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>12</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 75 12pt &quot;微软雅黑&quot;;
	
	color: rgb(0, 0, 0);
    background-color: rgba(255, 255, 255, 60%)
}</string>
       </property>
       <property name="text">
        <string>提交</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="feedback.qrc"/>
 </resources>
 <connections/>
</ui>
